person.rb

```diff
@@ -2,7 +2,7 @@
 
 class Person < Nameable
   # getter and accessor methods
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
Same here, @rentals should be set under `attr_accessor` because it must have both Getter and Setter.

- `attr_reader` makes it `read-only`, thus only a Getter method is applied.